xun/functions/util.py

```diff
@@ -420,6 +420,7 @@ def is_assignments_and_expressions(node):
     return all(
         isinstance(node, ast.Assign)
         or isinstance(node, ast.Expression)
+        or isinstance(node, ast.Expr)
         for node in node.body
     )
```

Comment:
It's possible to check all types at once. Also, I think the bug here was using `ast.Expression` in the first place, I _think_ it should've been `ast.Expr` all along

```suggestion
    return all(isinstance(node, (ast.Assign, ast.Expr)) for node in node.body)
```